src/org/traccar/protocol/AustinNbProtocolDecoder.java

```diff
@@ -57,11 +63,16 @@ protected Object decode(
         Position position = new Position(getProtocolName());
         position.setDeviceId(deviceSession.getDeviceId());
 
-        position.setTime(parser.nextDateTime());
+        position.setTime(parser.nextDateTime(Parser.DateTimeFormat.YMD_HMS, TimeZone.getDefault().getID()));
 
         position.setValid(true);
-        position.setLongitude(Double.parseDouble(parser.next().replace(',', '.')));
         position.setLatitude(Double.parseDouble(parser.next().replace(',', '.')));
+        position.setLongitude(Double.parseDouble(parser.next().replace(',', '.')));
+        position.set("azimuth", parser.nextInt());
```

Comment:
What's this? Why is it not assigned to course properly?

Reply:
An extra info we need for the protocol.

Reply:
This is exactly what course is. It's a properly in position model. You should use it instead.